enzyme/tools/enzyme-tblgen/blas-tblgen.cpp

```diff
@@ -739,7 +739,7 @@ void emit_extract_calls(const TGPattern &pattern, raw_ostream &os) {
 }
 
 // Will be used by Julia
-SmallString<80> ValueType_helper(const TGPattern &pattern, ssize_t actPos,
+SmallString<80> ValueType_helper(const TGPattern &pattern, size_t actPos,
```

Comment:
This isn't valid since actpos could be negative

Reply:
Seemed to not flag after reverting